ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.3.9f1
```

Comment:
I would probably leave this one as-is (or at least not downgrade the editor version).

I need to chat with the others to see how this managed otherwise.